mod 3 vignettes change to ref = develop

wokenny13 opened the pull request:
also update section on blank output for TADA_DefineCriteriaMethodology where no arg input should be needed

### Pull Request Checklist

-   [x] Ensure that your branch is up-to-date with the `develop` branch. Check for any merge conflicts and resolve them if needed.
  
-   [x] Include a summary of the changes made and any relevant context in the pull request description.

-   [ ] Link any related issues or pull requests. Use keywords like "Closes #issue_number" to automatically close related issues when the pull request is merged.

-   [ ] Document all code using line/inline and/or multi-line/block comments to describe what it does.
    
-   [x] Create or edit the function documentation. Include working examples. Run `devtools::document()`.

-   [ ] Update or add the new functionality to the appropriate vignette (or create a new one). Make sure the vignette is included in the articles section of the `_pkgdown.yml`.

-   [ ] Run spelling and styler maintenance in `requiredMaintenance.R`.

-   [ ] If function/code edits made as part of this issue impact other functions in the package or functionality in the shiny app, ensure those are updated as well.

-   [ ] Run `.TADA_UpdateRefFiles()` and `.TADA_UpdateExampleData()` in `MaintenanceScheduled.R`. All example data files need to be documented in `ExampleData.R` and also included in the `MaintenanceScheduled.R` so they get re-generated correctly during the scheduled routine maintenance (see `maintenance-scheduled.yaml`).

-   [ ] If you created any new columns or made changes, update `RequiredCols.R`.

-   [ ] Ensure that any new dependencies are added to the `DESCRIPTION` file and documented appropriately.

-   [ ] Create or edit tests in the `tests/testthat` folder to help prevent and/or troubleshoot potential future issues. Ensure that all new and existing tests pass after your changes. Run `devtools::test()` and verify.

-   [ ] Run `devtools::check()` and address any new notes or issues before creating a pull request.

-   [x] Request a review from at least one other developer on the team.

wokenny13 committed:
mod 3 change to ref = develop

also update section on blank output for TADA_DefineCriteriaMethodology where no arg input should be needed

cristinamullin commented on vignettes/ExampleMod3CriteriaMethodsAltOptions.Rmd:
````diff
@@ -71,7 +71,7 @@ may be prompted in the console to update dependencies that have more
 update all of them (enter 1 into the console).
 
-```{r install_TADA, eval = F, results = 'hide'}
+```{r install_TADA, eval = T, results = 'hide'}
````
Currently, it is installing TADA twice. Once finalized, only one of the code chunks (either install_TADA or dev_install) should have eval = T. If eval is not specified, like in dev_install, it defaults to eval = T.

wokenny13 replied:
In dev_install, the ref section there is commented out. I tried to follow consistency with one of the other vignettes (I think mod2). I am open to suggestions on what everyone else prefers to what each vignette does to install from develop once it is merged into develop.

wokenny13 committed:
make consistent installation of TADA throughout vignettes

For all vignettes:

set install_TADA eval = F

set dev_install eval = F

load
library(EPATADA)

wokenny13 committed:
duplicate chunk name fix in vignette TADAAssessmentUnitUseCase

wokenny13 commented:
Setting both to eval = F while having library(EPATADA) seems like it worked fine for the vignettes:

```
{r install_TADA, eval = F, results = 'hide'}
remotes::install_github("USEPA/EPATADA",
  ref = "develop",
  dependencies = TRUE
)
```
 
```
{r dev_install, eval = F, results = 'hide', include = F}
remotes::install_github("USEPA/EPATADA",
  ref = "develop",
  dependencies = TRUE
)
```
